src/main/java/usecase/alter_entry/AlterEntryController.java

```diff
@@ -0,0 +1,13 @@
+package usecase.alter_entry;
+
+public class AlterEntryController {
+    private final AlterEntryInputPort useCase;
+
+    public AlterEntryController(AlterEntryInputPort useCase) {
+        this.useCase = useCase;
+    }
+
+    public void alterEntry(int entry_id, Object new_value, String alter_column) {
```

Comment:
Convention: Please use a consistent style. In java, we used lower camelcase as normal variable and method names.
For example, `entry_id` should be changed to `entryId`.

You need to change **all** of your variable names, including the variables that follow.